Automattic/wp-calypso
# Rewind: Add signup flow for collecting Rewind credentials

Solves https://github.com/Automattic/wp-calypso/issues/18832

This PR adds an elaborate post-purchase flow for Jetpack plans, whose goal is to collect Rewind credentials.

**Testing instructions**
1. Load the branch in local dev.
2. Navigate to `http://calypso.localhost:3000/start/rewind-credentials?blogid=123456789`, obviously using a blog id which corresponds to a rewind-enabled Jetpack site.
3. Navigate through the flow and test all steps and step sequences. Note that you will need to re-navigate to the url in step 2 each time. Reloading your browser will not work because the flow will then lack the blog id context.
4. Attempt to enter credentials in the form and observe that they save.
